manifests/proxydhcp.pp

```diff
@@ -49,6 +49,12 @@
     failover       => $failover,
   }
 
+  exec { 'setfacl_dhcp':
+    command => 'setfacl -R -m u:theforeman:rwx /etc/dhcp /var/lib/dhcpd',
+    path => '/usr/bin',
+    onlyif => 'test -f /var/lib/dhcp && test -f /etc/dhcp',
```

Comment:
This is going to require some bash magic me thinks given the output of getfacl is:

```
ehelms@war puppet-foreman_proxy (fixes-20683)$ getfacl /etc/dhcp
getfacl: Removing leading '/' from absolute path names
# file: etc/dhcp
# owner: root
# group: root
user::rwx
group::r-x
other::---
```